Dockerfile

```diff
@@ -60,48 +60,37 @@ VOLUME ["/mnt"]
 # Build and install rizin on master branch
 # Remove all build dependencies
 # Cleanup
-RUN DEBIAN_FRONTEND=noninteractive dpkg --add-architecture i386 && \
-  apt-get update && \
-  apt-get install -y \
-  curl \
-  wget \
-  gcc \
-  git \
-  bison \
-  pkg-config \
```

Comment:
I think this is needed?

Reply:
bison, pkg-config are not used when building with meson (or at leasts it builds and the rizin binary seems  ok).

I only install those packages :  gcc  git python3-pip.

Do you have a use case where bison and pkg-config are used to build when using meson ?

Reply:
Then it's good :) I thought it meson/ninja needed that, but probably it's just to build against the libraries.